IndexedDB/interfaces.html

```diff
@@ -11,28 +11,35 @@
 
 <script>
 "use strict";
-setup(function() {
-  var idlArray = new IdlArray();
-
+async_test(function(t) {
   var request = new XMLHttpRequest();
   request.open("GET", "interfaces.idl");
   request.send();
   request.onload = function() {
```

Comment:
Needs to be wrapped in t.step_func in case of exceptions within, or t.step_func_done to save one line.